app/Models/Solo/Score.php

```diff
@@ -65,4 +68,65 @@ public function complete(array $params): void
 
         $this->save();
     }
+
+    public function createLegacyEntry()
+    {
+        $statAttrs = [
+            'Good',
+            'Great',
+            'LargeTickHit',
+            'LargeTickMiss',
+            'Meh',
+            'Miss',
+            'Ok',
+            'Perfect',
+            'SmallTickMiss',
```

Comment:
are these attributes going to be pascal cased in the final? (and thus not match `snake_case` everywhere else for our json)

Reply:
That's just our c# naming conventions coming through. We can probably change them if they are to be handled by `osu-web` going forward (although not sure we'd do the same if they were handled by a queue processor which was already .NET based..? ie. are the snake_casing conventions something we do in all cases for json, or when the receiving end is `osu-web`?)